themes/docs-new/layouts/partials/swiftype_search_modal.html

```diff
@@ -0,0 +1,73 @@
+    </div>
+
+    <div id="swiftype-lower-container" class="grid-container fluid">
+      <div id="swiftype-modal-content-container" class="grid-x grid-margin-x">
+        <!-- LEFT SIDEBAR -->
+        <div id="swiftype-container-left-sidebar" class="cell small-12 medium-shrink">
+          <button id="swiftype-custom-facets-toggle-button">
+            <h3>Chef Product
+              <span id="swiftype-custom-facets-toggle-caret" class="show-for-small-only">
+                <i class="fas fa-caret-down"></i>
+                <i class="fas fa-caret-up" style="display: none;"></i>
+              </span>
+            </h3>
+          </button>
+          <div id="swiftype-custom-facets">
+            <ul>
+              <li>
+                <input type="checkbox" class="product-filter" name="chef-product" id="swiftype-facet-automate" value="automate">
+                <label for="swiftype-facet-automate">Automate</label><br />
+              </li>
+              <li>
+                <input type="checkbox" class="product-filter" name="chef-product" id="swiftype-facet-desktop" value="desktop">
+                <label for="swiftype-facet-desktop">Desktop</label><br />
+              </li>
+              <li>
+                <input type="checkbox" class="product-filter" name="chef-product" id="swiftype-facet-habitat" value="habitat">
+                <label for="swiftype-facet-habitat">Habitat</label><br />
+              </li>
+              <li>
+                <input type="checkbox" class="product-filter" id="swiftype-facet-infra">
+                <label for="swiftype-facet-infra">Infra</label><br />
+                <ul>
+                  <li>
+                    <input type="checkbox" class="product-filter" name="chef-product" id="swiftype-facet-client" value="client">
+                    <label for="swiftype-facet-client">Infra Client</label><br />
+                  </li>
+                  <li>
+                    <input type="checkbox" class="product-filter" name="chef-product" id="swiftype-facet-server" value="server">
+                    <label for="swiftype-facet-server">Infra Server</label><br />
+                  </li>
+                </ul>
+              </li>
+              <li>
+                <input type="checkbox" class="product-filter" name="chef-product" id="swiftype-facet-inspec" value="inspec">
+                <label for="swiftype-facet-inspec">InSpec</label><br />
+              </li>
+              <li>
+                <input type="checkbox" class="product-filter" name="chef-product" id="swiftype-facet-workstation" value="workstation">
+                <label for="swiftype-facet-workstation">Workstation</label><br />
+              </li>
+            </ul>
+          </div>
+        </div>
+        <!-- RESULTS CONTAINER -->
+        <div id="swiftype-results-container" class="cell auto">
+          <div id="swiftype-results-heading"><h3>Search Results</h3></div>
+          <div id="swiftype-search-results">
```

Comment:
I'd suggest making the results a `<ul>` element populated with `<li>`'s.  Its better for accessibility and screen readers.

Reply:
What's interesting is that Swiftype's default search uses a div with a series of <a> tags, but no list. 🤷

Reply:
Interestingly enough, Google's search results aren't in a list form either.  I'd still argue that a list is a better use case, but at the end of the day these are accessibility guidelines that are still very loose and as long as we have a good reason for the choices we make, All is well in my opinion.